src/bin/nydusd/daemon.rs

```diff
@@ -293,9 +293,6 @@ pub trait NydusDaemon: DaemonStateMachineSubscriber {
     // NOTE: This method is not thread-safe, however, it is acceptable as
     // mount/umount/remount/restore_mount is invoked from single thread in FSM
     fn mount(&self, cmd: FsBackendMountCmd) -> DaemonResult<()> {
-        if self.backend_from_mountpoint(&cmd.mountpoint)?.is_some() {
```

Comment:
Why did we check the existence before? cc @changweige

Reply:
We want nydus to enforce a certain rule to handle APIs from clients.
It's not easy to justified whether extra MOUNT API request is misbehaved  or not from clients.
Moreover, I didn't think `fuse-backend-rs` crate had a clear semantics for this behavior then.

@luodw Shall we try to umount and than mount again?

Reply:
Agree, the mount is a POST API, its semantic should also not guarantee idempotency.

Reply:
ok，umount and than mount again also meet requirement.